docs/apiref/core.rst

```diff
@@ -11,3 +11,5 @@ DeepPavlov Core
    core.common <core/common>
    core.data <core/data>
    core.models <core/models>
+   core.agent <core/agent>
```

Comment:
What do you think of renaming deeppavlov.core.agent -> deeppavlov.core.agents (deeppavlov.core.skill -> deeppavlov.core.skills for consintency with deeppavlov.agents and deeppavlov.skills?

Reply:
For now, each file contains single core abstraction. I don't think that we should rename at least for now.

Reply:
I mean, in core we have agent_, while outside we have agentS.